best-practice-secure-comms.md

```diff
@@ -497,7 +497,7 @@ Certificate Revocation List (CRL) Profile"
 [OWASP-REST]: https://www.owasp.org/index.php/REST_Security_Cheat_Sheet
```

Comment:
Also needs fixing, to https://cheatsheetseries.owasp.org/cheatsheets/REST_Security_Cheat_Sheet.html

Reply:
I've taken the liberty of doing that